metabase/metabase
# Migrate ResetPassword form to formik

Epic https://github.com/metabase/metabase/issues/26178

As with other formik migration PRs, one of the added benefits is that it's now possible to fully unit test the form without complex mocking.

How to test:
- Setup email in Admin -> Settings
- Logout, then on the login screen click forgot your password, enter your email, and submit the form
- Click on the password reset link in the email
- You should be navigated to the ResetPassword form
- Make sure it shows error message for common passwords, like `admin123`
- Make sure it is possible to reset the password with the form

<img width="541" alt="Screenshot 2022-11-02 at 16 15 22" src="https://user-images.githubusercontent.com/8542534/199512769-86b8261c-8278-4636-9f72-3d53eb2cc5b2.png">
